assets/src/js/vwo-anti-flicker.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Initializes a global VWO (Visual Website Optimizer) queue function and ensures visibility for VWO elements.
+ *
+ * @return {{vwoCss: Function}} An object with a `vwoCss()` method stub.
+ */
+
+window.vwo_$ = window.vwo_$ || function() {
+  (window._vwoQueue = window._vwoQueue || []).push(arguments);
+  return {
+    vwoCss() {},
+  };
+};
+
+// eslint-disable-next-line no-undef
+vwo_$('body').vwoCss({'visibility':'visible !important'});
```

Comment:
This line does not seem to get added to the head tag and this is important to prevent the flicker issue.

Also, you will need to add this branch to the GPI dev site and ask Molly to test so as to make sure it still works as expected